calcom/cal.com
# Fix:use typed query initially fill default paramaters

## What does this PR do?
Default query params not being filled when loading a page with useTypedQuery.

Comparing prod vs this PR
https://github.com/calcom/cal.com/assets/55134778/ac2e6624-ff07-49c3-acb6-5f5110cbabbd

## Type of change




## How should this be tested?



- Load event types settings on dev and notice the default tab is already selected

## Mandatory Tasks
